modin/backends/base/query_compiler.py

```diff
@@ -1740,218 +3577,1371 @@ def set_index_names(self, names, axis=0):
+        and apply passed function to it.
+
+        Parameters
+        ----------
+        resample_args : list
+            Resample parameters in the format of ``modin.pandas.DataFrame.resample`` signature.
+        func : callable(pandas.Resampler) -> object
+        *args : iterable
+            Positional arguments to pass to function.
+        **kwargs : dict
+            Keyword arguments to pass to function.
+
+        Returns
+        -------
+        BaseQueryCompiler
+            New QueryCompiler containing the result of passed function.
+        """
+        return ResampleDefault.register(pandas.core.resample.Resampler.pipe)(
+            self, resample_args, func, *args, **kwargs
+        )
+
+    @_doc_resample_reduction(
+        result="product",
+        params="""_method : str
+        min_count : int""",
+        link="prod",
+    )
+    def resample_prod(self, resample_args, _method, min_count, *args, **kwargs):
+        return ResampleDefault.register(pandas.core.resample.Resampler.prod)(
+            self, resample_args, _method, min_count, *args, **kwargs
+        )
+
+    @_doc_resample_reduction(result="quantile", params="q : float", link="quantile")
+    def resample_quantile(self, resample_args, q, *args, **kwargs):
+        return ResampleDefault.register(pandas.core.resample.Resampler.quantile)(
+            self, resample_args, q, *args, **kwargs
+        )
+
+    @_doc_resample_reduction(
+        result="standart error of the mean", params="ddof : int, default: 1", link="sem"
+    )
+    def resample_sem(self, resample_args, ddof=1, *args, **kwargs):
+        return ResampleDefault.register(pandas.core.resample.Resampler.sem)(
+            self, resample_args, ddof, *args, **kwargs
+        )
+
+    @_doc_resample_reduction(result="number of element in a group", link="size")
+    def resample_size(self, resample_args, *args, **kwargs):
+        return ResampleDefault.register(pandas.core.resample.Resampler.size)(
+            self, resample_args, *args, **kwargs
+        )
+
+    @_doc_resample_reduction(
+        result="standart deviation", params="ddof : int", link="std"
+    )
+    def resample_std(self, resample_args, ddof, *args, **kwargs):
+        return ResampleDefault.register(pandas.core.resample.Resampler.std)(
+            self, resample_args, ddof, *args, **kwargs
+        )
+
+    @_doc_resample_reduction(
+        result="sum",
+        params="""_method : str
+        min_count : int""",
+        link="sum",
+    )
+    def resample_sum(self, resample_args, _method, min_count, *args, **kwargs):
+        return ResampleDefault.register(pandas.core.resample.Resampler.sum)(
+            self, resample_args, _method, min_count, *args, **kwargs
+        )
+
+    # FIXME: This method require us to build high-level resampler object
+    # which we shouldn't do at the backend. We need to move this at the front.
+    def resample_transform(self, resample_args, arg, *args, **kwargs):
```

Comment:
what's the difference between `resample_pipe` and `resample_transform`? their docstrings seem to be equal

Reply:
accidentally copy-pasted its doc-string)
Now `resample_transform` has the proper doc-string